### PR TITLE
Gives ability to buckle yourself to a toilet

### PR DESCRIPTION
**Pull request by chefferz**
I don't think this needs more explanation.

### CODE REVIEW

**Comment by ghost** [comment]
this is very uncanny

**Comment by chefferz** [comment]
> 
> 
> this is very uncanny

if you look at the branch name you might find a easter egg to why this is being done preemptively!!?

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -5,6 +5,8 @@
 	icon_state = "toilet00"
 	density = FALSE
 	anchored = TRUE
+	can_buckle = 1
```
Comment (silicons) [inline code/game/objects/structures/watercloset.dm]: 1 --> TRUE

**Comment by necromanceranne** [comment]
What, you need a seatbelt so you don't fall off and shit yourself?

File: code/game/objects/structures/watercloset.dm (hunk 1 above)
Comment (chefferz) [reply code/game/objects/structures/watercloset.dm]: I tested this locally. Can_buckle has to equal 1.

Comment (chefferz) [reply code/game/objects/structures/watercloset.dm]: Reference > https://github.com/Citadel-Station-13/Citadel-Station-13/blob/f2fc50119d33b36d1daccf33d2fc103f44c95ab9/code/game/objects/structures/beds_chairs/chair.dm

**Review by Ghommie — APPROVED** [review]
I doubt the branch name is a reference to space vacuum toilets... maybe Spaceball?

**Pull request merged**